packages/optimizers/terser/src/TerserOptimizer.js

```diff
@@ -19,23 +18,24 @@ export default new Optimizer({
       );
     }
 
-    let userConfig = await loadConfig(bundle.filePath, [
-      '.terserrc',
-      '.uglifyrc',
-      '.uglifyrc.js',
-      '.terserrc.js'
-    ]);
+    let userConfig = await bundle
+      .getEntryAssets()[0]
```

Comment:
Is this safe to do? When does a (javascript) bundle have multiple entry assets?

Reply:
Why the change?